.github/workflows/ci.yaml

```diff
@@ -12,7 +12,8 @@ concurrency:
 jobs:
   ci:
     name: ci
-    runs-on: ubuntu-latest
+    runs-on: org-${{ github.repository_owner_id }}-amd64-k8s
+    container: registry.suse.com/bci/bci-base:latest
```

Comment:
you can use ubuntu:22.04 as a container image but you will be required to manually install dependencies like github cli